packages/nodes-base/nodes/EmailSend/v2/send.operation.ts

```diff
@@ -220,6 +262,29 @@ export async function execute(this: IExecuteFunctions): Promise<INodeExecutionDa
 				mailOptions.html = this.getNodeParameter('html', itemIndex, '');
 			}
 
+			const appendAttribution = options.appendAttribution !== false && nodeVersion >= 2.1;
```

Comment:
If someone on version 2 would enable this option, it would not work right? (I think this suggestion works, but it depends on how `default` works on a node parameters). Alternatively you could hide the option for v2?

```suggestion
			const appendAttribution = options.appendAttribution === true
```

Reply:
thanks @elsmr ,  I pushed a fix